src/pages/Overview/Terms/Terms.jsx

```diff
@@ -0,0 +1,109 @@
+import React, { Component } from 'react'
+import { Route, Switch } from 'react-router-dom'
+
+//import NavigationBar from '../NavigationBar/NavigationBar'
+//import { stickyFooter } from '../Footer/Footer'
+import './Terms.css'
+
+class TermsItem extends Component {
+  render() {
+    return (
+      <div className='TermsItem' style={{ float: this.props.float }}>
+        {this.props.text}
+      </div>
+    )
+  }
+}
+
+class Terms extends Component {
+/*
+  componentDidMount() {
+    document.title = 'Terms and Conditions - TEN'
+    footerDelay()
+  }
+
+  componentDidUpdate() {
+    footerDelay()
+  }
+*/
```

Comment:
same here with comments and unused components

Reply:
updated for next PR!